showpass/Showpass-E2E-tests
# Updating the 'Checkout' spec

**Description:**
Previously another event was used for the 'Checkout' spec. Currently, we have a new event called "Test Automation event 3 (DO NOT MODIFY)".
This new event will be used in this spec going forward.

**How to test:**
Run the following test and ensure it passes.
cypress/e2e/checkout/checkout.cy.js

**Test execution results:**

<img width="947" alt="CheckoutTestUpdates" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/d6fd6518-837e-4bae-8b9c-5bf76a580cb8">
